apps/web-app/components/settings/addNewGroup.vue

```diff
@@ -405,9 +405,7 @@
             <UnUiInput
               v-model:value="newEmailIdentitySendNameValue"
               v-model:valid="newEmailIdentitySendNameValid"
-              v-model:validationMessage="
-                newEmailIdentitySendNameValidationMessage
-              "
+              v-model:validationMessage="newEmailIdentitySendNameValidationMessage"
```

Comment:
this wont work as prettier will wrap it to a new line next time the file is saved
might be better to rename it to `newEmailSendNameValidationMessage` so its below the wrap character limit